### DISCUSSION TITLE
Ideias para TCC

### DISCUSSION THREAD

**Discussion by dricoosz** [frontendbr/forum#1198] [Discussão]
Pessoal, estou no meu último ano de sistemas da informação, preciso fazer um TCC e gostaria de fazer sobre metodologia ágil, mas metodologia ágil é algo bem amplo, pensei em fazer um comparativo entre as metodologia tradicionais e metodologias ágeis, mas meu orientador me disse que é algo já manjado.
Aqui no grupo não achei nada relacionado a esse tema, poderiam me dar sugestões, problemas que existam hoje, que eu poderia trabalhar meu TCC em cima das questões.

Aqueles que puderem ajudar agradeço, e aquele que não, também agradeço!!

**Comment by felipefialho** [comment]
Vê se ajuda  => #529 #621 #524

**Comment by dricoosz** [comment]
Olá @LFeh, muito obrigado pelo retorno.

Já tinha dado uma olhada no 524, agora dei uma consultada no 621, mas não encontrei algo que pudesse me ajudar, estava querendo fazer algo mais voltado para área de planejamento de projetos, gerenciamento e afins...

Mas alguma recomendação?

**Comment by tunnes** [comment]
@dricoosz Cara já usou o trello seguindo a [metodologia lean](https://leankit.com/learn/lean/lean-methodology/)? Um problema era a visualização da relação entre tarefas, pois o trello provê uma visualização vertical e a relação futura entre as tarefas do sprint não. Mesmo implementando componentes e classes reutilizaveis e abstratas você as vezes era pego de supresa com o seguinte problema: "Se eu soubesse que essa feature iria ficar assim, na tarefa do sprint anterior eu teria implementado diferente", um fluxograma de tarefas com integração com a API do trello, ajudaria bastante, pois o gestor iria organizar esse diagrama e o dev poderia ter uma previsão de como vai ficar a feature nos próximos sprints.

**Comment by carloap** [comment]
Olá @dricoosz o tema parece ser bem complexo e amplo.
Então, talvez o "Ágil" que você vá abordar descreva as tendências do DevOps, e do Lean IT com beneficio para cultura organizacional e que tbm virá como "suporte" a eles e muito mais na ITIL v4 (2019) - ou seja, tudo está utilizando ágil.

Acredito que os principais problemas sejam: como pensar na melhor solução no início, no valor retornado, reduzir o retrabalho, evitar desperdícios, etc. E suas possíveis soluções: Usar o KanBan (ex: trello), blindar o time (filtrar demandas de desperdício), motivar, usar ferramentas (ex: SIPOC), seguir o pipeline do projeto e manter a melhora contínua (Scrum)

Se ajuda, veja sobre as metodologias Six Sigma, Kaizen, DMAIC

**Comment by dricoosz** [comment]
@tunnes e @carloap, dei uma pesquisada sobre metodologia lean, até então não à conhecia, e achei bem interessante. 
Acho interessante ver a caminhada da abordagem como DevOps. Uso bastante o trello, mas somente para projetos e planejamentos pessoais. 
Mas fiquei na dúvida sobre qual abordagem o tema talvez fosse abordar.

Agradeço ai pelas sugestões, se puderem continuar a me direcionar nesse assunto.... agradeço!
